eo-maven-plugin/src/main/java/org/eolang/maven/SafeMojo.java

```diff
@@ -104,11 +107,28 @@ abstract class SafeMojo extends AbstractMojo {
     @Parameter(property = "eo.scope")
     protected String scope = "compile";
 
+    /**
+     * Cached tojos.
+     * @checkstyle VisibilityModifierCheck (5 lines)
+     */
+    protected final Unchecked<Tojos> tojos = new Unchecked<>(
+        new Sticky<>(
+            () -> new Catalog(this.foreign.toPath(), this.foreignFormat).make()
+        )
+    );
+
     @Override
     public final void execute() throws MojoFailureException {
         StaticLoggerBinder.getSingleton().setMavenLog(this.getLog());
         try {
+            final StopWatch times = new StopWatch();
             this.exec();
+            Logger.info(
```

Comment:
@mximp I would put it to `debug` logging scope, otherwise it may look like noise in the log

Reply:
@yegor256 Don't we want to see this in normal build just like timings for tests:
```
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.013 s - in EOorg.EOeolang.EOor_with_two_negTest
```